pages/services/data-science-engine/1.0.0/spark/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: layout.pug
+navigationTitle: Spark
+excerpt: DC/OS Data Science Engine and Spark integration
+title: Spark
+menuWeight: 8
+model: /services/data-science-engine/data.yml
+render: mustache
+---
+
+{{ model.techName }} comes with Apache Spark integration and allows running Spark jobs from {{ model.nickName }} notebooks and terminal.
+
+# Launching a Spark job
+## Using Terminal
+Open a `Terminal` from Notebook UI and run example `spark-submit` job:
```

Comment:
```suggestion
Open a `Terminal` the from Notebook UI and run this example `spark-submit` job:
```

Reply:
Not sure I understand "the from Notebook". I think you mean "from the"? On that assumption, I have made the changes you suggest.